rm-datasource/src/main/java/io/seata/rm/datasource/undo/mysql/MySQLUndoUpdateExecutor.java

```diff
@@ -58,10 +58,9 @@ protected String buildUndoSQL() {
         Field pkField = row.primaryKeys().get(0);
         List<Field> nonPkFields = row.nonPrimaryKeys();
         String updateColumns = nonPkFields.stream()
-            .map(field -> keywordChecker.checkAndReplace(field.getName()) + " = ?")
+            .map(field -> field.getName() + " = ?")
             .collect(Collectors.joining(", "));
-        return String.format(UPDATE_SQL_TEMPLATE, keywordChecker.checkAndReplace(sqlUndoLog.getTableName()),
-                             updateColumns, keywordChecker.checkAndReplace(pkField.getName()));
+        return String.format(UPDATE_SQL_TEMPLATE, sqlUndoLog.getTableName(), updateColumns, pkField.getName());
```

Comment:
I think we need to escape character judgment here on the primary key.
![image](https://user-images.githubusercontent.com/8758457/74300856-16b5e100-4d8c-11ea-87fd-4b46aed01a28.png)

Reply:
![image](https://user-images.githubusercontent.com/8758457/74304448-fa1fa600-4d97-11ea-9f86-091bc4e1167a.png)